torch/csrc/jit/python/python_ir.cpp

```diff
@@ -780,6 +780,12 @@ void initPythonIRBindings(PyObject* module_) {
         return get_python_cu()->get_interface(
             c10::QualifiedName(qualified_name));
       }))
+      .def(
+          "getMethod",
+          [](InterfaceType& self, const std::string& name) {
+            return self.getMethod(name);
+          },
+          py::return_value_policy::reference)
```

Comment:
This means the lifetime of the returned `method` object is managed by C++? And it will be alive until the CU is gone?

Reply:
Correct. 
Notice `self.getMethod(name)` returns a raw pointer. 

If not passed by reference, pybind11 creates a unique_ptr wrapping the raw pointer.

So Python GC will `delete` the raw pointer when destructing the unique_ptr, causing later accesses from C++ world using a dangling pointer.